component/mcsdadmin/component.go

```diff
@@ -404,30 +414,134 @@ func newEndpointPost(w http.ResponseWriter, r *http.Request) {
 			http.Error(w, "bad request: could not find organization", http.StatusBadRequest)
 			return
 		}
+
+		var epRef fhir.Reference
+		epRef.Type = to.Ptr("Endpoint")
+		epRef.Reference = to.Ptr("Endpoint/" + *resEp.Id)
+		epRef.Display = to.Ptr(endpoint.Address)
```

Comment:
I'd not do this, since if the address changes, it's now out of sync. And it makes people integrate against display, because then they don't have to lookup the endpoint.